### ISSUE TITLE
Accessibility guideline tests should work in flutter run environment

### ISSUE THREAD

**Issue by jonahwilliams** [flutter/flutter#32057]
Initially because the elements we are looking for are not hitTestable, and then because we seemingly do not find the correct Rect. 

cc @HansMuller

**Comment by goderbauer** [comment]
@jonahwilliams Do you still have context on what was meant by this issue? Not clear to me if this is still actionable...

**Comment by jonahwilliams** [comment]
If you have a test that makes an assertion using https://github.com/flutter/flutter/blob/83720015a44e1c7c665cd77d6f1501e41e7ceaa4/packages/flutter_test/lib/src/accessibility.dart#L113 , then that test will seemingly always fail if you use `flutter run` instead of `flutter test`

**Comment by chunhtai** [comment]
What is the use case for running guideline test in flutter run environment? lowering priority because I think most case they will be running in Flutter Tester environment
